src/sentry/processing/backpressure/health.py

```diff
@@ -0,0 +1,61 @@
+from typing import Mapping
+
+import sentry_sdk
+from django.conf import settings
+
+from sentry import options
+from sentry.processing.backpressure.topology import CONSUMERS
+from sentry.utils import redis
+
+
+def _prefix_key(key_name: str) -> str:
+    return f"bp1:{key_name}"
+
+
+UNHEALTHY_KEY_NAME = "unhealthy-consumers"
+
+
+def _unhealthy_consumer_key(name: str) -> str:
+    return _prefix_key(f"{UNHEALTHY_KEY_NAME}:{name}")
+
+
+service_monitoring_cluster = redis.redis_clusters.get(
+    settings.SENTRY_SERVICE_MONITORING_REDIS_CLUSTER
+)
```

Comment:
For a follow-up: Let's use a client with a low, strict timeout. Probably, `socket_timeout=1` would be applicable here. 

Hanging on this call isn't problematic because of the delay -- we would halt the consumer anyway -- but because arroyo needs to continue polling. Therefore, we need to ensure that we cannot suspend the caller for too long.

Reply:
> I think a way to do this is to structure the setting similar to [this](https://github.com/getsentry/sentry/blob/97b8674540984c84b6e65c258759fc5c9e88ed05/src/sentry/conf/server.py#L3312-L3316). Then, always create a dedicated client that connects to the cluster with the supplied options.

Is that link going where it's meant to? That option is never used and doesn't seem to have anything to do with Redis.